intercom_flutter/pubspec.yaml

```diff
@@ -1,12 +1,13 @@
 name: intercom_flutter
 description: Flutter plugin for Intercom integration. Provides in-app messaging
   and help-center Intercom services
-version: 3.1.0
+version: 3.2.0
```

Comment:
Since this is a big change (adding web implementation + plugin interface) I think we can bump it to `4.0.0`

Reply:
ok, will bump the version to 4.0.0 when we integrate intercom_flutter_web in intercom_flutter